tasks/util/container_commands.js

```diff
@@ -15,6 +15,7 @@ containerCommands.ping = [
 containerCommands.setup = [
     containerCommands.cpIndex,
     containerCommands.restart,
+    'sleep 1',
```

Comment:
Looks like the `npm run pretest` is a little more flaky than the previous list of shell commands on CIrcleCi. 

Not sure what's up.

I've added sleep delay:
- between the `nw` restart and the ping as well as as
- between the docker run and setup commands. 

Hopefully that will fix it. But we might have to look into this further.